### PR TITLE
Switch position of Comments and Pagination sections and change the ap…

### LINKED ISSUE #3572: Pagination popup impedes pagination
### Problem
If the pagination of a process is conducted by the images alone, it is not possible to: 
- browse the images 
- to see the pagination information

### Solution
The pagination popup should be flexible, so that the images can be browsed and examined. 

### Example
paginierung01

paginierung02

### PR DESCRIPTION
**Pull request by IkramMaalej**
…pearance of the latter from a popup to a section

Fixes #3572

### CODE REVIEW

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/dialogs/pagination.xhtml [hunk 1]
```diff
@@ -12,177 +12,147 @@
...
+                        </div>
+                        <div>
+                            <p:outputLabel value="#{msgs.counting}:"
+                                           for="selectPaginationMode"/>
+                            <p:selectOneMenu
+                                    value="#{DataEditorForm.paginationPanel.selectPaginationModeSelectedItem}"
+                                    converter="#{IllustratedSelectItemConverter}"
+                                    panelStyle="width:25%;"
+                                    var="mode"
+                                    id="selectPaginationMode">
```
Comment (solth) [inline Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/dialogs/pagination.xhtml:89]:
```suggestion
                                    var="mode"
                                    autoWidth="false"
                                    id="selectPaginationMode">
```

**Review by solth — APPROVED** [review]
Could you rename "Fiktive Paginierung" to "Fingierte Paginierung" in `messages_de.properties`?

The two buttons "Cancel" and "Apply" in German are to wide for the default width of the second column:
Bildschirmfoto 2021-03-02 um 13 18 47

**Comment by IkramMaalej** [comment]
> Could you rename "Fiktive Paginierung" to "Fingierte Paginierung" in `messages_de.properties`?
> 
> The two buttons "Cancel" and "Apply" in German are to wide for the default width of the second column:
> Bildschirmfoto 2021-03-02 um 13 18 47

For me the buttons problem is not reproducible in either Chrome or Safari.

**Comment by solth** [comment]
> For me the buttons problem is not reproducible in either Chrome or Safari.

I used Firefox

**Comment by Kathrin-Huber** [comment]
Please rebase to fix seleniumtests

**Comment by IkramMaalej** [comment]
> > For me the buttons problem is not reproducible in either Chrome or Safari.
> 
> I used Firefox

I don't have this problem also with Firefox

**Comment by solth** [comment]
> I don't have this problem also with Firefox

I think we can fix this in a separate PR.

**Pull request merged**